alacritty/src/config/bindings.rs

```diff
@@ -327,6 +327,8 @@ pub enum ViAction {
     InlineSearchNext,
     /// Jump to the previous inline search match.
     InlineSearchPrevious,
+    ///Star search
+    StarSearch,
```

Comment:
Since you can rebind things it doesn't make sense to name them after their binding. It should be named after what it does.

```suggestion
    /// Start forward search for the word at the current cursor position.
    SearchCurrentWordForward,
```